client/enduser/src/main/resources/META-INF/resources/app/js/app.js

```diff
@@ -1,3 +1,5 @@
+/* global $http, AuthService */
```

Comment:
Is this required? If not, please remove; otherwise move it past the license header.